.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@LuosNahim Please exclude the **.vs** folder and all its contents from source control